### ISSUE TITLE
Rpak loading issues

### ISSUE THREAD

**Issue by NachosChipeados** [R2Northstar/NorthstarLauncher#821]
I tried updating some of my mods to use the new rpak loading structure but i found some issues:

Setting the loading option to "." or "mp_" causes the rpak to not get loaded at all.
![Screenshot 2024-09-10 203617](https://github.com/user-attachments/assets/8728c7a0-6c5f-4dbc-a07c-d857dcc1e28a)
![Screenshot 2024-09-10 203628](https://github.com/user-attachments/assets/6893c3f0-ffc6-4692-bb85-a5383dd85fcf)
![Screenshot 2024-09-10 203643](https://github.com/user-attachments/assets/b8241216-8231-4f64-9286-31a6f56e065e)

Setting it to use a specific map causes the textures to load, but become invisible (though this doesn't happen with the example satchel mod)
![Screenshot 2024-09-10 203722](https://github.com/user-attachments/assets/570acde7-478c-4ca2-bc10-1198ac87e455)
![Screenshot 2024-09-10 203733](https://github.com/user-attachments/assets/2fe8c1f6-8735-4e06-a1a2-1fc3bd83f82f)
![Screenshot 2024-09-10 203747](https://github.com/user-attachments/assets/52152ac0-8682-40f9-b8b6-dba32d4a5614)

Using the old structure (postload) works completely fine.
![Screenshot 2024-09-10 204206](https://github.com/user-attachments/assets/d0752db4-bfcf-40bb-b99d-cfbf1676f2af)
![Screenshot 2024-09-10 204214](https://github.com/user-attachments/assets/24c81343-40e2-4af1-86d9-57e71cfe99f4)

**Comment by NachosChipeados** [comment]
I set up three mods using the models above to quickly test this:

The first one (AAA)  uses the old loading method
The second one (BBB) uses "."
The third one (CCC) uses "mp_angel_city"

https://www.mediafire.com/file/93jbubqtro8cgu8/dwdadwada.zip/file

(Uploading to mediafire because of githubs file size limit)

**Comment by GeckoEidechse** [comment]
cc @ASpoonPlaysGames

**Comment by ASpoonPlaysGames** [comment]
Yeah, I can repro this... not really sure why it happens though. I'll need to spend some time reversing to look at what the difference between loading common and loading map rpaks is.

**Comment by ASpoonPlaysGames** [comment]
So the difference between `common.rpak` and our ones is with the flags it seems. `common.rpak` gets a flags value of `0x2` but ours get a flags value of `0x7` when loading. Going to try giving ours the same flags as `common.rpak` and see what happens

**Comment by ASpoonPlaysGames** [comment]
Tested various things, not sure I can do much about this. It seems to be to do with *when* the rpak is loaded in some capacity. Even using Postload with a different rpak results in the invisibility issue. I guess some of this stuff will have to remain using Postload for now?
